db/migrate/20220121100200_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :bio
```

Comment:
- [ ] `:bio` should be of type `text`. And not of type `string`. It will be nice to fix this.

Reply:
This issue is well resolved! ✔️